blog/fama-french-three-vs-five-factors/index.qmd

```diff
@@ -0,0 +1,82 @@
+---
+title: "Comparing Fama-French Three vs Five Factors"
+author:
+  - name: Christoph Scheuch
+    url: https://christophscheuch.github.io/
+    affiliations:
+      - name: wikifolio Financial Technologies AG
+date: "2023-09-20"
+description: An explanation for the difference in the value factors of FF3 and FF5 data
+image: thumbnail.png
+image-alt: Two stylized high rise office buildings, one small building and one large building, both buildings visible in the frame. The background has a futuristic setting, with a backdrop of a digital, grid-like landscape symbolizing the internet. The color palette should be a combination of cool blues and warm yellow. Created with DALL-E 2.
+categories: 
+  - Data
+  - Replication
+--- 
+
+In our book chapter [Replicating Fama and French Factors](../../r/replicating-fama-and-french-factors), we show how to construct factor portfolios that are fairly close to the popular data from [Prof. Kenneth French finance data library](https://mba.tuck.dartmouth.edu/pages/faculty/ken.french/data_library.html). In this blog post, I want to elaborate a bit more on the subtle differences between the size and value data in the Fama-French three (FF3) and five (FF5) factor data. 
```

Comment:
Maybe add the two paper citations as footnotes.

Would it make sense to investigate Google search requests for the differences between the factors for SEO?

Reply:
We hardly get any impressions for fama french replication, so I'd not overinvest for now.